dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/scheduling/DefaultJobConfigurationService.java

```diff
@@ -192,4 +210,22 @@ private String prettyPrint( String field )
 
         return String.join( " ", fieldStrings );
     }
+
+    private JobParameters getDefaultJobParameters( JobConfiguration jobConfiguration )
+    {
+        if ( jobConfiguration.getJobType().getJobParameters() == null )
+        {
+            return null;
+        }
+
+        try
+        {
+            return jobConfiguration.getJobType().getJobParameters().newInstance();
+        }
+        catch ( InstantiationException | IllegalAccessException ignored )
```

Comment:
Could you add a short comment about why we are ignoring these exceptions?

Reply:
Added debug error log.